assets/jsx/bagger.jsx

```diff
@@ -17,8 +17,19 @@ class Bagger extends React.Component {
     }
 
     componentDidMount() {
-        const { dispatch, uploader: {accessKeyId, secretAccessKey, bucket, region}} = this.props
-        dispatch(BagActions.testConfiguration(accessKeyId, secretAccessKey, bucket, region))
+        const { dispatch } = this.props
+        const b = document.getElementById('bagger');
+        if (b.dataset.accessKeyId !== undefined) { // TODO: any? all? add fine grain actions?
```

Comment:
Thoughts on what this guard should look like? Check that they are all set if any of them are set and throw an error if not. Or maybe should we have individual guards with corresponding fine grain actions? Or...